mmcv/ops/csrc/common/cuda/nattenav_cuda_kernel.cuh

```diff
@@ -0,0 +1,148 @@
+/*!
+**************************************************************************************************
+* NATTEN-COMMON FUNCTIONS (CUDA)
+* Copyright (c) 2020 SenseTime. All Rights Reserved.
+* Licensed under the Apache License, Version 2.0 [see LICENSE for details]
+**************************************************************************************************
+* Modified from
+*https://github.com/SHI-Labs/Neighborhood-Attention-Transformer
+**************************************************************************************************
+*/
+#include <ATen/ATen.h>
```

Comment:
We don't prefer to include torch dependency in `csrc/common` since it might be shared by other backend without torch.

Reply:
ATen dependency has been removed from `nattenav_cuda_kernel.cuh`